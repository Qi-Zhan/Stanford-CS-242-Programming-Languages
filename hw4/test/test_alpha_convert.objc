START VERIFY;
# Test that alpha conversion is being performed.
x = [f1 = \o.o];
[f2 = \o. x.f1].f2;
STOP VERIFY;
