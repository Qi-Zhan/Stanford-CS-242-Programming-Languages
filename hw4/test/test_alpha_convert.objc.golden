x = [f1 = \o.(o)];
[f1 = \o_0.(o_0)];
